### PR TITLE
[release] v0.6.6

### PR DESCRIPTION
**Pull request by yuanming-hu**
Related issue = #677 

Added a major contributor list of Simplified Chinese documentation: https://github.com/taichi-dev/taichi/blob/7bf13aaec3f3d441b39aacc059385e8688ad6e59/README.md#developers

Thank you so much for contributing the Chinese translation, which is extremely helpful for people whose native language is not English. Special thanks to @StephenArk30 and @archibate, for building the translation infrastructure, organizing & reviewing the translation process!

[[Click here for the format server]](http://kun.csail.mit.edu:31415/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,9 @@ The Taichi project was created by [Yuanming Hu (yuanming-hu)](https://github.com
 
 [Kenneth Lozes (KLozes)](https://github.com/KLozes) and [Yu Fang (squarefk)](https://github.com/squarefk) have also made notable contributions. [[List of all contributors]](https://github.com/taichi-dev/taichi/graphs/contributors)
 
+The Simplified Chinese documentation (简体中文文档) is mainly contributed by [Ark (StephenArk30)](https://github.com/StephenArk30), [彭于斌 (archibate)](https://github.com/archibate), [Danni (isdanni)](https://github.com/isdanni),
+[Rex (rexwangcc)](https://github.com/rexwangcc), and [jianzhouwan (ArkhamWJZ)](https://github.com/ArkhamWJZ). [[List of all contributors]](https://github.com/taichi-dev/taichi-docs-zh-cn/graphs/contributors).
+
```
Comment (archibate) [inline README.md:62]: How about to follow the form above, use `created by` for @StephenArk30 just like in L53?
```suggestion
The Simplified Chinese documentation (简体中文文档) is created by [Ark (StephenArk30)](https://github.com/StephenArk30). Significant contributions are made by:
  - [彭于斌 (archibate)](https://github.com/archibate)
  - [Danni (isdanni)](https://github.com/isdanni)
  - [Rex (rexwangcc)](https://github.com/rexwangcc)
  - [jianzhouwan (ArkhamWJZ)](https://github.com/ArkhamWJZ)
  
  [[List of all contributors]](https://github.com/taichi-dev/taichi-docs-zh-cn/graphs/contributors).
```
```

Comment (yuanming-hu) [reply README.md:62]: Thank you! Sounds good!

File: README.md [hunk 2]
```diff
@@ -57,10 +57,18 @@ The Taichi project was created by [Yuanming Hu (yuanming-hu)](https://github.com
...
+The Simplified Chinese documentation (简体中文文档) was created by [Ark (StephenArk30)](https://github.com/StephenArk30). Significant contributions are made by:
+  - [彭于斌 (archibate)](https://github.com/archibate)
+  - [Danni (isdanni)](https://github.com/isdanni)
+  - [Rex (rexwangcc)](https://github.com/rexwangcc)
+  - [jianzhouwan (ArkhamWJZ)](https://github.com/ArkhamWJZ)
+  - [[List of all contributors]](https://github.com/taichi-dev/taichi-docs-zh-cn/graphs/contributors).
+
+-------------------------------
+
+If you would like to contribute to Taichi, **please check out our [Contributor Guidelines](https://taichi.readthedocs.io/en/latest/contributor_guide.html).**
```
Comment (archibate) [inline README.md:74]: Why this is gone?
```suggestion
We welcome feedback and comments. If you would like to contribute to Taichi, **please check out our [Contributor Guidelines](https://taichi.readthedocs.io/en/latest/contributor_guide.html).**
```

Comment (yuanming-hu) [reply README.md:74]: I feel like our active maintenance already implies that *We welcome feedback and comments.* So we don't really need this...

Comment (archibate) [reply README.md:74]: Simply put `If you would like to contribute to Taichi` sounds rude to me. It sounds like persuading people to contribute to us as volunteer, with no welcome provided.

Comment (yuanming-hu) [reply README.md:74]: Sounds good. Let's keep the old version.

File: README.md [hunk 3]
```diff
@@ -55,7 +55,21 @@ The Taichi project was created by [Yuanming Hu (yuanming-hu)](https://github.com
...
+[Kenneth Lozes (KLozes)](https://github.com/KLozes) and [Yu Fang (squarefk)](https://github.com/squarefk) have also made notable contributions.
+
+[[List of all contributors to Taichi]](https://github.com/taichi-dev/taichi/graphs/contributors)
+
+-------------------------------
+
+The Simplified Chinese documentation (简体中文文档) was created by [Ark (StephenArk30)](https://github.com/StephenArk30). Significant contributions are made by:
+  - [彭于斌 (archibate)](https://github.com/archibate)
+  - [Danni (isdanni)](https://github.com/isdanni)
+  - [Rex (rexwangcc)](https://github.com/rexwangcc)
```
Comment (rexwangcc) [inline README.md:67]:
```suggestion
  - [Chengchen Wang (rexwangcc)](https://github.com/rexwangcc)
```
Seems like folks are using their full names here, maybe I should do it too...

File: README.md [hunk 4]
```diff
@@ -55,7 +55,21 @@ The Taichi project was created by [Yuanming Hu (yuanming-hu)](https://github.com
...
+  - [jianzhouwan (ArkhamWJZ)](https://github.com/ArkhamWJZ)
+
+[[List of all contributors to the Simplified Chinese documentation of Taichi]](https://github.com/taichi-dev/taichi-docs-zh-cn/graphs/contributors)
```
Comment (archibate) [inline README.md:70]: Btw, how about to move this list to taichi-docs-zh-cn/README.md? So no this long desc needed.

File: README.md [hunk 5]
```diff
@@ -55,7 +55,21 @@ The Taichi project was created by [Yuanming Hu (yuanming-hu)](https://github.com
  - [彭于斌 (archibate)](https://github.com/archibate) (OpenGL Compute Shader backend)
  - [Mingkuan Xu (xumingkuan)](https://github.com/xumingkuan) (IR optimization & standardization)
 
-[Kenneth Lozes (KLozes)](https://github.com/KLozes) and [Yu Fang (squarefk)](https://github.com/squarefk) have also made notable contributions. [[List of all contributors]](https://github.com/taichi-dev/taichi/graphs/contributors)
+[Kenneth Lozes (KLozes)](https://github.com/KLozes) and [Yu Fang (squarefk)](https://github.com/squarefk) have also made notable contributions.
```
Comment (archibate) [inline README.md:58]: Why not move KLozes and squarefk to the list above too?

File: README.md (hunk 3 above)
Comment (isdanni) [reply README.md:67]: same haha

File: README.md [hunk 6]
```diff
@@ -55,7 +55,21 @@ The Taichi project was created by [Yuanming Hu (yuanming-hu)](https://github.com
...
+
+[[List of all contributors to Taichi]](https://github.com/taichi-dev/taichi/graphs/contributors)
+
+-------------------------------
+
+The Simplified Chinese documentation (简体中文文档) was created by [Ark (StephenArk30)](https://github.com/StephenArk30). Significant contributions are made by:
+  - [彭于斌 (archibate)](https://github.com/archibate)
+  - [Danni (isdanni)](https://github.com/isdanni)
```
Comment (isdanni) [inline README.md:66]: - [Danni Li (isdanni)](https://github.com/isdanni)

File: README.md (hunk 5 above)
Comment (isdanni) [reply README.md:58]: second this

File: README.md (hunk 6 above)
Comment (isdanni) [inline README.md:66]: - [Danni Li (isdanni)](https://github.com/isdanni)

File: README.md (hunk 5 above)
Comment (KLozes) [reply README.md:58]: No feelings hurt here! You guys on the above lust have done ton more work than me!

**Comment by archibate** [comment]
Thank for all your valuable discussions, but I noticed that the `[release]` PR is becoming a `[readme]` PR.
I would suggest to merge this for now without README updates, keep this PR focus on release, and move the how-to-show-contributions-in-README-md-problem to another PR.

File: README.md (hunk 4 above)
Comment (ArkhamWJZ) [reply README.md]: Good idea ;)  Thanks for your guys amazing work! I'm just doing what I can (>▽<)
Btw, may I use my Chinese name?
- 万健洲(ArkhamWJZ)

File: README.md (hunk 5 above)
Comment (yuanming-hu) [reply README.md]: Thanks for the proposal and for understanding. I actually think, for now, the existing description achieves a good trade-off between *visibility* and *fairness*.

Of course, the list will be updated in the future whenever necessary.

**Comment by yuanming-hu** [comment]
> Thank for all your valuable discussions, but I have to say that the `[release]` PR is becoming a `[readme]` PR, which may not be @yuanming-hu's initial intention.
> I would suggest to merge this for now without README updates, keep this PR focus on release, and move the how-to-show-contributions-in-README-md-problem to another PR.

Let's keep the README update in this PR given people already provided their feedback and (mostly) reached a consensus here.

File: README.md (hunk 4 above)
Comment (yuanming-hu) [reply README.md:70]: I think we should keep the list here for more visibility. Of course, feel free to maintain a more detailed version there if you want.

**Pull request merged**